packages/123done/oauth.js

```diff
@@ -296,8 +296,12 @@ module.exports = function (app, db) {
                 }
               );
             })
-            .catch(function (err) {
-              return res.send(400, err);
+            .catch(function () {
+              return res
+                .status(400)
+                .send(
+                  'Error: Client secret mismatch, please verify secret or obtain from an Admin.'
```

Comment:
That's the only possible error in this catch?

Reply:
I'll add check if its a JWT validation error, those are almost always secret mismatch